diff -up meshlab-1.3.2/vcglib/wrap/io_tetramesh/import_ply.h.cstddef meshlab-1.3.2/vcglib/wrap/io_tetramesh/import_ply.h
--- meshlab-1.3.2/vcglib/wrap/io_tetramesh/import_ply.h.cstddef	2011-01-25 00:53:39.000000000 -0700
+++ meshlab-1.3.2/vcglib/wrap/io_tetramesh/import_ply.h	2012-09-30 02:40:19.697251228 -0600
@@ -36,6 +36,7 @@ created
 #include<wrap/ply/io_mask.h>
 #include<wrap/io_tetramesh/io_ply.h>
 #include<vcg/complex/tetramesh/allocate.h>
+#include <cstddef>
 
 
 
diff -up meshlab-1.3.2/vcglib/wrap/io_trimesh/import_ply.h.cstddef meshlab-1.3.2/vcglib/wrap/io_trimesh/import_ply.h
--- meshlab-1.3.2/vcglib/wrap/io_trimesh/import_ply.h.cstddef	2012-06-27 04:07:33.000000000 -0600
+++ meshlab-1.3.2/vcglib/wrap/io_trimesh/import_ply.h	2012-09-30 02:40:19.699251228 -0600
@@ -29,6 +29,7 @@
 #include<wrap/io_trimesh/io_mask.h>
 #include<wrap/io_trimesh/io_ply.h>
 #include<vcg/complex/algorithms/create/platonic.h>
+#include<cstddef>
 
 namespace vcg {
 namespace tri {
diff -up meshlab-1.3.2/vcglib/wrap/ply/plystuff.h.cstddef meshlab-1.3.2/vcglib/wrap/ply/plystuff.h
--- meshlab-1.3.2/vcglib/wrap/ply/plystuff.h.cstddef	2011-01-25 00:53:42.000000000 -0700
+++ meshlab-1.3.2/vcglib/wrap/ply/plystuff.h	2012-09-30 02:49:41.399295985 -0600
@@ -57,6 +57,8 @@ Cleaning of the automatic bbox caching s
 #include <sys/types.h>
 #include <sys/stat.h>
 #include <fcntl.h> 
+#include <cstddef>
+#include <unistd.h>
 #ifdef WIN32
 #include <io.h>
 #endif
